src/index.html

```diff
@@ -5,8 +5,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/reseter.css/1.0.8/reseter.min.css">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
```

Comment:
maybe here better to name the body class something like "page"

Reply:
But if I will make some page, I ill have a problem there))))